aws/corehandlers/user_agent.go

```diff
@@ -19,13 +19,13 @@ var SDKVersionUserAgentHandler = request.NamedHandler{
 const execEnvVar = `AWS_EXECUTION_ENV`
 const execEnvUAKey = `exec_env`
 
-// AddHostExecEnvUserAgentHander is a request handler appending the SDK's
+// AddHostExecEnvUserAgentHandler is a request handler appending the SDK's
 // execution environment to the user agent.
 //
 // If the environment variable AWS_EXECUTION_ENV is set, its value will be
 // appended to the user agent string.
-var AddHostExecEnvUserAgentHander = request.NamedHandler{
-	Name: "core.AddHostExecEnvUserAgentHander",
+var AddHostExecEnvUserAgentHandler = request.NamedHandler{
```

Comment:
Regrettably changing the name of this handler will be a breaking change in the SDK.